File: server_integration_tests/clients/dart_upload_client/test/fixtures.dart
import 'package:graphlink_server_integration_tests_dart_upload_client/generated/graphlink.dart';

const _port = 9996;

String get httpUrl => 'http://localhost:$_port/graphql';

GraphLinkClient newClient() => GraphLinkClient.withHttp(url: httpUrl);

File: server_integration_tests/clients/ts_client/test/fixtures.ts
import { GraphLinkClient } from '../src/generated/client/graph-link-client.js';
import { DefaultGraphLinkWsAdapter } from '../src/generated/client/graph-link-default-ws-adapter.js';
import { createFetchAdapter } from '../src/generated/client/graph-link-adapters.js';

const port = parseInt(process.env.SERVER_PORT || '9997', 10);
export const httpUrl = `http://localhost:${port}/graphql`;
export const wsUrl = `ws://localhost:${port}/graphql`;

export function newClient(): GraphLinkClient {
  return new GraphLinkClient(
    createFetchAdapter(httpUrl),
    new DefaultGraphLinkWsAdapter(wsUrl),
  );
}
